nvflare/lighter/project.yml

```diff
@@ -2,17 +2,6 @@ api_version: 3
 name: example_project
 description: NVIDIA FLARE sample project yaml file
 
-client_components: &cln_comps
-  resource_manager:    # This id is reserved by system.  Do not change it.
-    path: nvflare.app_common.resource_managers.list_resource_manager.ListResourceManager
-    args:
-      resources:
-        gpu: [0, 1, 2, 3]
-  resource_consumer:    # This id is reserved by system.  Do not change it.
-    path: nvflare.app_common.resource_consumers.gpu_resource_consumer.GPUResourceConsumer
-    args:
-      gpu_resource_key: gpu
-  
```

Comment:
I am thinking the components section should still exist, just that the arguments will be changed to something like 
```
args:
    resource_config: "site.json"
```

Reply:
Let's keep the component's section in this PR, I will modify these places along with changes to resource_managers and resource_consumers

Reply:
OK.